src/components/Message/MessageSimple/MessageActionSheet.tsx

```diff
@@ -97,6 +117,7 @@ export type MessageActionSheetProps = {
    * React useState hook setter function that toggles action sheet visibility
    */
   setActionSheetVisible: React.Dispatch<React.SetStateAction<boolean>>;
+  supportedReactions: Array<unknown>;
```

Comment:
should be an `Reaction[]` with `Reaction` being imported from `ReactionList`

Reply:
Will update :)